www/docs/en/dev/guide/platforms/android/index.md

```diff
@@ -32,276 +32,152 @@ development. For a comparison of the two development paths, see the
 
 ## Requirements and Support
 
+### Java Development Kit (JDK)
+
+Install [Java Development Kit (JDK) 7](http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html)
+or later.
+
+When installing on Windows you also need to set `JAVA_HOME` Environment Variable
```

Comment:
I'm not 100% sure, but this is someting that should be set by `check_reqs` at the [build time](https://github.com/apache/cordova-android/blob/master/bin/lib/check_reqs.js#L99)